java-extension/com.microsoft.java.test.plugin/src/main/java/com/microsoft/java/test/plugin/util/RuntimeClassPathUtils.java

```diff
@@ -49,9 +52,10 @@
         for (final IPath testPath : testPaths) {
             final Iterator<IJavaProject> iterator = javaProjectSet.iterator();
             while (iterator.hasNext()) {
-                final IJavaProject project = iterator.next();
-                if (project.getProject().getLocation().isPrefixOf(testPath)) {
-                    projectsToTest.add(project);
+                final IJavaProject javaProject = iterator.next();
+                final IProject project = javaProject.getProject();
+                if (belongToVisibleProject(testPath, project) || belongToInvisibleProject(testPath, project)) {
```

Comment:
Combine these to API. 

Just include in Workspace in okay

Reply:
Updated